shared/src/commonMain/kotlin/io/customer/shared/tracking/constant/MetricEvent.kt

```diff
@@ -0,0 +1,18 @@
+package io.customer.shared.tracking.constant
+
+import kotlinx.serialization.SerialName
+
+/**
+ * Metric event enum to determine possible metric events.
+ */
+@kotlinx.serialization.Serializable
+enum class MetricEvent {
```

Comment:
I think it is great to keep things at module level possible. But I would not want to combine it with KMM. We can keep it as a separate activity so that we don't complicate things at one time. Also, this might be required by both `MessagingPush` and `MessagingInApp`, so perhaps it is best to keep it in the common module for now.